modin/pandas/test/test_io.py

```diff
@@ -1052,6 +1052,16 @@ def test_from_csv_s3(make_csv_file):
     df_equals(modin_df, pandas_df)
 
 
+def test_read_parquet_s3():
+    import s3fs
+
+    dataset_url = "aws-roda-hcls-datalake/chembl_27/chembl_27_public_tissue_dictionary/part-00000-66508102-96fa-4fd9-a0fd-5bc072a74293-c000.snappy.parquet"
+    fs = s3fs.S3FileSystem(anon=True)
+    pandas_df = pandas.read_parquet(fs.open(dataset_url, "rb"))
+    modin_df = pd.read_parquet(fs.open(dataset_url, "rb"))
```

Comment:
S3-links should be used here as in the reproducer for #1765

Reply:
S3-links is supported by `pandas.read_parquet` only when default credentials for `boto` are available (pandas 1.1.3). To make testing possible I applied this way when I make anonymous credentials using `S3FileSystem(anon=True)`. 

When pandas will release `storage_options` for `read_*` functions we should include this feature for modin (tracker for this #2301). After that we can rewrite test by next way:
```python
dataset_url = "s3://bucket/path/*.parquet"
pandas_df = pandas.read_parquet(dataset_url, storage_options = {'anon': True})
modin_df = pd.read_parquet(dataset_url, storage_options = {'anon': True})
df_equals(pandas_df, modin_df_s3fs)
```

In current version I've implemented internal resolving between anonymous and default boto parameters like it's done in `read_csv`, for example. Tests for both cases were added.

Reply:
Great, thanks!